README.md

```diff
@@ -1,4 +1,186 @@
+   in your browser, you can drag-and-drop the image from your local file
+   explorer into the folder in the browser window.
+
+6. Add your post contents. Read about Markdown features in
+   [the official Docusaurus Markdown guide](https://docusaurus.io/docs/markdown-features).
+
+7. Every post needs a preview section. This preview section appears as the
+   summary of the blog post, and there's a "read more" link underneath. To mark
+   where the preview section ends, add the following as a separate line:
+
+   ```html
+   <!--truncate-->
+   ```
+
+8. We use Code Spell Checker (cSpell). If you use unfamiliar words in your post,
+   cSpell will underline them. If it's actually a word you want to use, hover
+   over it with your mouse and select **Quick actions**, and select **Add to
+   workspace settings**. This saves the word to be used in other posts and by
+   other users of the project.
+
+   **HINT:** Instead of hovering with your mouse, you can press Ctrl+P or Cmd+P.
+
+9. If you want to add images, in the folder where you created your `index.md`
+   file, create an `images` folder and place them there. Remember that file
+   paths in this environment are **case sensitive**.
+
+   If you are working in VS Code in the browser, you can drag-and-drop images
+   from your file explorer into the browser window.
+
+## Creating a PR
+
+When you're done editing, it's time to create a **Pull Request (PR)**. Your
+teammates can review the PR and give you helpful hints. The PR will also trigger
+a validation build which can help you catch problems with broken links, etc.
+
+1. Stage your changes. In VS Code, switch to the **Version control** tab (the
+   little Git icon, looks like a graph). Hover over the list of changed files.
+   If you hover over the top part, of the list, you should see a **+** (plus)
+   icon. When you click it, all files are staged.
+2. In the box above the list of staged files, type in the commit message. make
+   is descriptive.
+3. Click **Synchronize** (or Push).
+4. Once your changes are pushed, VS Code will pop up a message to create a Pull
+   Request. Click it and follow the onscreen guide.
+
+   Don't forget to add reviewers by clicking **Request review**.
+
+You can also create a PR in GitHub. Navigate to the root of the repo. You should
+see an alert at the top which tells you pushed a new branch and you can create a
+pull request.
+
+### Delete the branch after merge
+
+Once your PR is approved and the checks have passed, merge it and delete the
+branch. Thanks~!
+
+## Setting up a GitHub codespace
+
+The easiest way to contribute to the site, is to set up a GitHub codespace. This
+allows you to run VSCode in the browser, run a site preview, create branches and
+commits, etc.
+
+You can save a codespace and keep it for future use, so the procedure below is
+one-time only.
+
+To set up a new codespace:
+
+1. In GitHub, click your user avatar and select **Your Codespaces**.
```

Comment:
Not sure if it's me but I don't see 'Your codespaces' option to select.

Reply:
Widziałem, że niektórzy nie dostają tej opcji.

Dodam uwagę, że trzeba wejść na https://github.com/codespaces